fdbclient/include/fdbclient/RunRYWTransaction.actor.h

```diff
@@ -53,6 +53,21 @@ Future<decltype(std::declval<Function>()(Reference<ReadYourWritesTransaction>())
 	}
 }
 
+ACTOR template <class Function>
+Future<Void> runRYWTransactionVoid(Database cx, Function func) {
```

Comment:
It might be useful to add a comment about when this function should be used.

Reply:
Done